linalg/sparsemat.cpp

```diff
@@ -28,6 +28,14 @@ namespace mfem
 
 using namespace std;
 
+void SparseMatrix::InitCuSparse()
+{
+   /* Initialize cusparse library */
+#ifdef MFEM_USE_CUDA
+   cusparseCreate(&handle);
```

Comment:
Why does this work when multiple `SparseMatrix` instances are constructed? I verified that it does work, but I am curious why this is not an issue.

Reply:
Are you thinking this only has to be done once? I can look into that, I think a static variable should do the trick. I don't think I can answer why it works for multiple sparse matrices though.

Reply:
I added a static variable to avoid calling cusparseCreate  for every sparse matrix. Only the first instance of sparse matrix will create it and the last will destroy it.